coalib/results/Diff.py

```diff
@@ -166,16 +166,21 @@ def from_clang_fixit(cls, fixit, file):
         """
         assert isinstance(file, (list, tuple))
 
-        oldvalue = '\n'.join(file[fixit.range.start.line-1:
-                                  fixit.range.end.line])
-        endindex = fixit.range.end.column - len(file[fixit.range.end.line-1])-1
-
-        newvalue = (oldvalue[:fixit.range.start.column-1] +
-                    fixit.value +
-                    oldvalue[endindex:])
-        new_file = (file[:fixit.range.start.line-1] +
-                    type(file)(newvalue.splitlines(True)) +
-                    file[fixit.range.end.line:])
+        try:
+            oldvalue = '\n'.join(file[fixit.range.start.line-1:
+                                      fixit.range.end.line])
+            endindex = fixit.range.end.column - \
+                len(file[fixit.range.end.line-1])-1
```

Comment:
please use braces instead of backslashes to break statements over multiple lines.